src/lib/i18n/locales/en-GB/translation.json

```diff
@@ -408,8 +408,8 @@
 	"Model not selected": "",
 	"Model Params": "",
 	"Model updated successfully": "",
-	"Model Whitelisting": "",
-	"Model(s) Whitelisted": "",
+	"Model Allowlisting": "",
```

Comment:
As mentioned in my previous comment, "Restrict model selection" would be a much better translation imo.